cmd/kubeadm/app/cmd/phases/util.go

```diff
@@ -31,3 +32,14 @@ func SetKubernetesVersion(cfg *kubeadmapiv1.ClusterConfiguration) {
 	}
 	cfg.KubernetesVersion = version.Get().String()
 }
+
+// CopyFileTo copy file from src to dest.
+func CopyFileTo(src string, dest string) error {
```

Comment:
suggesting we rename the function to just CopyFile e.g. as in this discussion:
https://github.com/golang/go/issues/8868

```suggestion
// CopyFile copy file from src to dest.
func CopyFile(src string, dest string) error {
```